### DISCUSSION TITLE
Should there be a Fall back mechanism to operate servers, if and when things go wrong with Teleport?

### DISCUSSION THREAD

**Discussion by alwaysastudent** [gravitational/teleport#9948] [Q&A] [answered]
Is there a fallback mechanism or any advice that we should implement if things go wrong with Teleport cluster? Would a Teleport Node agent fail at any point or in any circumstance, what are the chances that it can go offline? How can we guarantee that the Teleport Cluster is fault tolerant? 

Saw this tip from the official documentation, and it got us wondering.

> TIP
> In most environments, we advise replacing the OpenSSH daemon sshd with the Teleport Node Service unless there are existing workflows relying on ssh or in special cases such as embedded devices that can't run custom binaries.
Reference: https://goteleport.com/docs/architecture/nodes/

**Comment by webvictim** [comment, marked as answer]
IMO it's always advisable to maintain `sshd` for emergency access in case Teleport crashes for some reason or there's another issue. With the best will in the world, Teleport is not as well-tested or battle-hardened as `sshd` and I would far rather be relying on `sshd` in a true emergency.
